images/plbase/plbase-install.sh

```diff
@@ -61,18 +62,13 @@ arch=`uname -m`
 curl -LO https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-${arch}.sh
 bash Miniforge3-Linux-${arch}.sh -b -p /usr/local -f
 
-# R is not yet supported on ARM64. If we're on ARM64 or R package installation
-# is specifically disabled, we'll avoid installing anything R-related.
-if [[ "${arch}" != "aarch64"  ]] && [[ "${SKIP_R_PACKAGES}" != "yes" ]]; then
+# If R package installation is specifically disabled, we'll avoid installing anything R-related.
+if [[ "${SKIP_R_PACKAGES}" != "yes" ]]; then
     echo "installing R..."
-    conda install r-essentials
+    conda install --channel r r-essentials
 
     echo "installing Python packages..."
     python3 -m pip install --no-cache-dir -r /python-requirements.txt
-
-    echo "installing R packages..."
-    echo "set SKIP_R_PACKAGES=yes to skip this step"
-    Rscript /r-requirements.R
```

Comment:
Should we reach out to CEE 202 and ask them to check their questions?

Reply:
That sounds very reasonable, can you take that on?

Reply:
I ran the command below in the `pl-cee202` repo and confirmed that none of these packages are being referenced by any of their questions, so I think are good to go.

```
for m in tidyverse remotes RcppArmadillo rmarkdown RSQLite nycflights13 fueleconomy babynames rbenchmark microbenchmark maps maptools mapproj mapdata ggmap fivethirtyeight caret e1071 factoextra gbm glmnet ISLR kernlab klaR mlbench nnet pROC randomForest rpart rpart.plot rsample kableExtra quantreg ; do echo $m ; git grep $m ; done
```

Reply:
Great, thanks for checking!